src/core/GameKeyHandler.ttslua

```diff
@@ -57,6 +58,80 @@ function addDoomToAgenda()
   doomCounter.call("addVal", 1)
 end
 
+-- move the hovered object to the nearest empty slot on the playermat
+function takeCardIntoThreatArea(playerColor, hoveredObject)
+  -- only continue if an unlocked card
```

Comment:
I don't think you can even hover a locked object but I might be wrong (at least you don't get the yellow outline)